### ISSUE TITLE
[#SomeChanges Suggestion] Give DK Tanks AP Debuff

### ISSUE THREAD

**Issue by bscothern** [ClassicWoWCommunity/cata-classic-bugs#763]
### Give DK Tanks AP Debuff
In WotLK DKs are the only tanks that can't bring both the an attack speed and attack power debuff. In Cata this was fixed by making Blood Plague bring the attack power debuff. It would be a small but very nice improvement if there was a way this could be brought into the current version of the game.

This could be done via a minor glyph that applies the additional effect when in frost presence or by updating a talent to also bring it. If it is applied to a talent, ideally it would be something like Blade Barrier since it is a tank talent already and it is at the top of the tree so if someone wants the debuff as a frost or unholy tank they can get it still.

**Comment by Shrimp6387** [comment]
Yes, i would highly appreciate it. We always need to have a warrior, ret with spec into, feral jumping in for bear shout from time to time or have another OT. Imo its really annoying and at some bosses incredible... The idea with the Talent or Minor Glyph is very nice. I guess putting it in as Blade Barrier is the best idea so no dps can abuse it without specing into. Minor Glyph only might be to strong. Since AP debuff is so strong, please add this.

**Comment by JamminL** [comment]
Archiving old non-relevant issues in preparation for MOP. If I made a mistake please comment on it.
